jx_module/modules/cluster/main.tf

```diff
@@ -13,8 +13,7 @@ resource "azurerm_kubernetes_cluster" "aks" {
   }
 
   network_profile {
-    load_balancer_sku = "Basic"
-    network_plugin    = "kubenet"
+    network_plugin = "kubenet"
```

Comment:
kubenet is not the recommended network plugin for production. CNI should be

Reply:
Can you point me in the direction of the docs on best practice for this? Most clusters I work with we're using kubenet without issues. It would be good to know if these are sub-optimal implementations and why.

Reply:
The other issue I foresee with using CNI rather than kubenet is IP space proliferation. I think if we implement this it should be as an option to the user to specify the networking model. So perhaps we open a new PR where this is parameterised. Does that suit?

Reply:
https://docs.microsoft.com/en-us/azure/aks/concepts-network#compare-network-models

We had this discussion with the product engineers as well. 

Another good reference: https://www.youtube.com/watch?v=JyLtg_SJ1lo&feature=youtu.be

Reply:
On the grounds that the article you linked says `However, there are advantages and disadvantages to each` - I think we should go with making this the users choice?

Reply:
And if you're happy with that I don't see any harm with putting that in this PR

Reply:
Ok, just pushed a commit with this defaulted to kubenet (as AKS does out of the box) but the ability for users to override to  azure/cni